src/common/token/arweave.ts

```diff
@@ -27,9 +27,14 @@ import { sha256B64Url, toB64Url } from '../../utils/base64.js';
 import { sleep } from '../../utils/common.js';
 import { TurboWinstonLogger } from '../logger.js';
 
+const ArweaveClass =
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore -- Access the correct class constructor for Arweave
+  ArweaveModule.default?.default || ArweaveModule.default || ArweaveModule;
```

Comment:
nit: consider making this an export and using it where we want to use `Arweave`

Reply:
right now we just access it here and then in test env. using this exported version in our test env just causes compiler issues due to this being a const. as arweave-js exports right now, it seems like we need this pattern in each file that we try to access it